util/nrt_diff/nrtDiff.py

```diff
@@ -1210,6 +1219,116 @@ def DTWDistance(left, right) :
 
     return DTW[n][m]
 
+# Utility class to compare IIDM files
+class IIDMobject:
+    def __init__(self,ID):
+        self.id=ID
+        self.type=""
+        self.values = {}
+
+# Utility method to compare IIDM files
+def set_values(element,what,IIDMobject):
+    if element.hasAttribute(what):
+        IIDMobject.values[what] = element.getAttribute(what)
+
+# Read a IIDM file name and build a dictionary object id => values
+# Only values that can be changed by dynawo are taken into account
+def getOutputIIDMInfo(filename):
+    IIDM_objects_byID = {}
+    iidm_root = ImportXMLFile(filename)
+    for voltageLevel in iidm_root.getElementsByTagName("voltageLevel"):
```

Comment:
if powsybl iidm has prefix iidm: on all elements will it work ? It not we need to keep in mind that we may have to correct it later on

Reply:
I agree, this is the next step of this development